.gitattributes

```diff
@@ -1 +1,4 @@
 * text eol=lf
+*.png binary
+*.svg binary
```

Comment:
I do not understand the reason why .svg should be treated as binary, since SVG is text-based.

Reply:
You're absolutely right, it makes no sense. I think it was late. :see_no_evil: